src/pages/r4-page-channel-player.js

```diff
@@ -0,0 +1,22 @@
+import { html } from 'lit'
+import BaseChannel from './base-channel'
+
+export default class R4PageChannelPlayer extends BaseChannel {
+	render() {
+		if (!this.channel) return
+		return html`
+			<a href=${this.channelOrigin}>
+				<r4-avatar
+					slug=${this.channel.slug}
```

Comment:
```suggestion
					image=${this.channel.image}
```

if you pass it `slug`, it fetches the channel. If you pass it `image` (cloudinary id), it just displays